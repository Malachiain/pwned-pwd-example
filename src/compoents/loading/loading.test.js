import React, {Component} from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Loading from './index'
describe('The Loading page ',()=>{
    it('It renders the loading page',()=>{
        const wrapper = shallow(<Loading/>)
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})