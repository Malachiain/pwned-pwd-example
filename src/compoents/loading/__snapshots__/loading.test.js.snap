// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Loading page  It renders the loading page 1`] = `
<styled.div>
  <FontAwesomeIcon
    border={false}
    className=""
    fixedWidth={false}
    flip={null}
    icon="spinner"
    inverse={false}
    listItem={false}
    mask={null}
    pull={null}
    pulse={false}
    rotation={null}
    size="10x"
    spin={true}
    swapOpacity={false}
    symbol={false}
    title=""
    transform={null}
  />
</styled.div>
`;
